src/hidra/receiver/asapo_transfer.py

```diff
@@ -121,6 +107,20 @@ def main():
     run_transfer(asapo_transfer, args['reconnect_timeout'])
 
 
+def construct_config(config_file, identifier):
+
+    # Read config file
+    with open(config_file, "r") as f:
+        config = yaml.load(f.read(), Loader=yaml.FullLoader)
```

Comment:
It would be great to verify the config, e.g., check that all necessary keys and no unknown keys are present and that all value have to correct type.

Reply:
Function to verify config is added.

Reply:
Function to verify config is added.